### PR TITLE
[CTOR-1141]-Refresh network-ruggedcom doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] 25.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Comment by YanMeddour** [comment]
not sure about the prerequisites

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 1]
```diff
@@ -2,48 +2,516 @@
...
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Ruggedcom-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                | Modèle de service                              | Description                                                                                          | Découverte |
+|:---------------------|:-----------------------------------------------|:-----------------------------------------------------------------------------------------------------|:----------:|
+| Traffic-Generic-Id   | Net-Ruggedcom-Traffic-Generic-Id-SNMP-custom   | Contrôle de la bande passante de l'interface. Pour chaque contrôle apparaîtra le nom de l'interface  |            |
+| Traffic-Generic-Name | Net-Ruggedcom-Traffic-Generic-Name-SNMP-custom | Contrôle de la bande passante de l'interface. Pour chaque contrôle apparaîtra le nom de l'interface  |            |
+| Traffic-Global       | Net-Ruggedcom-Traffic-Global-SNMP-custom       | Contrôle de la bande passante de l'interface. Pour chaque contrôle apparaîtra le nom de l'interface  | X          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md:38]: no difference at all

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 2]
```diff
@@ -2,48 +2,516 @@
...
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte de services
+
+| Nom de la règle                 | Description                                                             |
+|:--------------------------------|:------------------------------------------------------------------------|
+| Net-Ruggedcom-SNMP-Traffic-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md:53]:
```suggestion
| Net-Ruggedcom-SNMP-Traffic-Name | Découvre les interfaces réseau et en supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 3]
```diff
@@ -2,48 +2,516 @@
...
+> Concerne les modèles de service suivants : Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+### Configuration SNMP
+
+Le service SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-ruggedcom.md:109]: port 161

File: pp/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 4]
```diff
@@ -2,48 +2,515 @@
...
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Ruggedcom-SNMP-custom" label="Net-Ruggedcom-SNMP-custom">
+
+| Service Alias   | Service Template                          | Service Description                          |
+|:----------------|:------------------------------------------|:---------------------------------------------|
+| Errors          | Net-Ruggedcom-Errors-SNMP-custom          | Check hardware errors                        |
+| Hardware-Global | Net-Ruggedcom-Hardware-Global-SNMP-custom | Check all sensors                            |
+| Memory          | Net-Ruggedcom-Memory-SNMP-custom          | Check the rate of the utilization of memory  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ruggedcom.md:25]:
```suggestion
| Memory          | Net-Ruggedcom-Memory-SNMP-custom          | Check the rate of memory utilization |
```

File: pp/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 5]
```diff
@@ -2,48 +2,515 @@
...
+| Temperature     | Net-Ruggedcom-Temperature-SNMP-custom     | Check hardware temperature                   |
+
+> The services listed above are created automatically when the **Net-Ruggedcom-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                               | Service Description                                                                          | Discovery  |
+|:---------------------|:-----------------------------------------------|:---------------------------------------------------------------------------------------------|:----------:|
+| Traffic-Generic-Id   | Net-Ruggedcom-Traffic-Generic-Id-SNMP-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ruggedcom.md:35]:
```suggestion
| Traffic-Generic-Id   | Net-Ruggedcom-Traffic-Generic-Id-SNMP-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear  |            |
```

File: pp/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 6]
```diff
@@ -2,48 +2,515 @@
...
+| Traffic-Generic-Name | Net-Ruggedcom-Traffic-Generic-Name-SNMP-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ruggedcom.md:36]:
```suggestion
| Traffic-Generic-Name | Net-Ruggedcom-Traffic-Generic-Name-SNMP-custom | Check the bandwidth of the interface. For each check the name of the interface will appear  |            |
```

File: pp/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 7]
```diff
@@ -2,48 +2,515 @@
...
+| Traffic-Global       | Net-Ruggedcom-Traffic-Global-SNMP-custom       | Check the bandwidth of the interface. For each checks the name of the interface will appear  | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ruggedcom.md:37]:
```suggestion
| Traffic-Global       | Net-Ruggedcom-Traffic-Global-SNMP-custom       | Check the bandwidth of the interface. For each check the name of the interface will appear  | X          |
```

File: pp/integrations/plugin-packs/procedures/network-ruggedcom.md [hunk 8]
```diff
@@ -2,48 +2,515 @@
...
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-### Centreon Plugin
+### SNMP Configuration
+
+The SNMP service must be configured and activated on the host. Please refer to the official documentation from the constructor/editor.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-ruggedcom.md:108]: port 161

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Pull request merged**